tests/UnifiedSpecTests/Operation.php

```diff
@@ -295,7 +295,12 @@ private function executeForCollection(Collection $collection)
                 assertIsArray($args['requests']);
 
                 return $collection->bulkWrite(
-                    array_map('self::prepareBulkWriteRequest', $args['requests']),
```

Comment:
Was this syntax deprecated in PHP 8.2?

Reply:
Yes, partially supported callables were deprecated in PHP 8.2, including any callable patterns that include `self`, `parent`, or `static`. [This page](https://php.watch/versions/8.2/partially-supported-callable-deprecation) contains the full list of deprecated callable syntaxes.